mobile/envoy_build_config/BUILD

```diff
@@ -56,6 +52,14 @@ envoy_cc_library(
             "@envoy//source/common/quic:quic_transport_socket_factory_lib",
         ],
         "@envoy",
+    ) + envoy_select_envoy_mobile_stats_reporting(
+        [
+            "@envoy//source/extensions/stat_sinks/metrics_service:config",
+            "@envoy//source/extensions/stat_sinks/statsd:config",
+            "@envoy//source/extensions/clusters/logical_dns:logical_dns_cluster_lib",
+            "@envoy//source/extensions/clusters/static:static_cluster_lib",
```

Comment:
This doesn't seem obviously related to stats, but I'm guessing this is used to implement the stats cluster?

Reply:
actually looks like this one can go in my TODO test only extension PR